explorer/deploy/webAppCreator.ts

```diff
@@ -543,4 +569,26 @@ class WebsiteStep extends WebAppCreatorStepBase {
         }
     }
 
+    //Implements new Service principal model for ACR container registries while maintaining old admin enabled use
+    private async acquireRegistryLoginCredentials(): Promise<void> {
+        if (this._serverPassword && this._serverUserName) { return; }
+
+        const client = new ContainerRegistryManagementClient(this.azureAccount.getCredentialByTenantId(this._imageSubscription.tenantId), this._imageSubscription.subscriptionId);
+        const resourceGroup: string = this._registry.id.slice(this._registry.id.search('resourceGroups/') + 'resourceGroups/'.length, this._registry.id.search('/providers/'));
+
+        if (this._registry.adminUserEnabled) {
+            try {
+                let creds = await client.registries.listCredentials(resourceGroup, this._registry.name);
```

Comment:
move this to `_util` file inside a new function `listRegistryCredentials `?

Reply:
While this makes sense to me conceptually I am honestly having trouble with how to go about it, I could create a function say in azureUtils that takes an azure account, a subscription, and a registry to do this and while this seems reasonable, and maybe I am biased because of PR #332 , it doesn't seem to follow the existing pattern of this file and seems more like something to add to my unified credentials that would look odd anywhere else. It seems most reasonable to me to add it to the Unified credentials branch and leave this like it is, at some point it may be good to refactor this code around that other branch however but that goes for the overall file,

Reply:
Sounds good, you can consider refactoring in the other branch. Also, you are right about the overall file in general.